lib/external/sentry.js

```diff
@@ -1,7 +1,19 @@
 const { inspect } = require('util');
 const { isBlank } = require('../util/util');
 
-const sensitiveEndpoints = ['/users/reset/verify', '/users/(.*?)/password'];
+const sensitiveEndpoints = [
+  '/users/reset/verify',
+  '/users/(.*?)/password',
+  '/users',
```

Comment:
Maybe be worth adding a comment mentioning that this is also intended to match /users/:id?